docs/network protocol.md

```diff
@@ -149,11 +150,11 @@ Sent to clients to acknowledge a received [LocationScouts](#LocationScouts) pack
 Sent when there is a need to update information about the present game session. Generally useful for async games.
 Once authenticated (received Connected), this may also contain data from Connected.
 #### Arguments
-The arguments for RoomUpdate are identical to [RoomInfo](#RoomInfo) barring:
+The arguments for RoomUpdate are identical to [RoomInfo](#RoomInfo), and can also contain arguments from
+[Connected](#Connected), barring:
```

Comment:
I know you added this because Beserker kept pushing it, but this is literly already written on line 151 (3 lines up)

Also the arguments arent identical to RoomUpdate as all arguments are optional, but thats already stated on line 162

Reply:
I honestly don't know what the ask is here.

Reply:
i meant the "and can also contain arguments from
[Connected](#Connected)" you added was already on there 3 lines up "this may also contain data from Connected."